### PR TITLE
Revert "Update comment in hunting monstergroups to mention copy-from"

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Corrects change that was incorrect. By removing comment .

#### Describe the solution
Deletes sentence.

#### Describe alternatives you've considered
None

#### Testing
Blah

#### Additional context

Reverts CleverRaven/Cataclysm-DDA#68058

### CODE REVIEW

**Comment by Maleclypse** [comment]
Needs to change the line “mods can overwrite” to “mods implicitly extend”

**Comment by RenechCDDA** [comment]
https://discord.com/channels/598523535169945603/598535827169083403/1150172481303351396

Just delete that comment entirely IMO. Looks like there's some general documenting to be done.

File: data/json/monstergroups/misc.json [hunk 1]
```diff
@@ -516,7 +516,7 @@
     "type": "monstergroup",
     "name": "GROUP_CAMP_HUNTING",
     "//": "Special group with hardcoded reference in basecamp::hunting_results. Does not actually spawn anywhere, used solely to determine returns from camp hunting mission.",
-    "//1": "Can be overwritten by mods.  Or extended by use of copy-from.",
+    "//1": "Can be overwritten by mods.",
```
Comment (Maleclypse) [inline data/json/monstergroups/misc.json:519]:
```suggestion
```

File: data/json/monstergroups/misc.json [hunk 2]
```diff
@@ -533,7 +533,7 @@
     "type": "monstergroup",
     "name": "GROUP_CAMP_TRAPPING",
     "//": "Special group with hardcoded reference in basecamp::hunting_results. Does not actually spawn anywhere, used solely to determine returns from camp hunting mission.",
-    "//1": "Can be overwritten by mods.  Or extended by use of copy-from.",
+    "//1": "Can be overwritten by mods.",
```
Comment (Maleclypse) [inline data/json/monstergroups/misc.json:536]:
```suggestion
```

File: data/json/monstergroups/misc.json [hunk 3]
```diff
@@ -555,7 +555,7 @@
     "type": "monstergroup",
     "name": "GROUP_CAMP_HUNTING_LARGE",
     "//": "Special group with hardcoded reference in basecamp::hunting_results. Does not actually spawn anywhere, used solely to determine returns from camp hunting mission.",
-    "//1": "Can be overwritten by mods.  Or extended by use of copy-from.",
+    "//1": "Can be overwritten by mods.",
```
Comment (Maleclypse) [inline data/json/monstergroups/misc.json:558]:
```suggestion
```

**Pull request merged**